oidc_provider/management/commands/creatersakey.py

```diff
@@ -9,8 +11,12 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         try:
-            key = RSA.generate(1024)
-            rsakey = RSAKey(key=key.exportKey('PEM').decode('utf8'))
+            key = generate_private_key(public_exponent=65537, key_size=2048, backend=default_backend())
+            rsakey = RSAKey(
+                key=key.private_bytes(
+                    encoding=serialization.Encoding.PEM,
+                    format=serialization.PrivateFormat.TraditionalOpenSSL,
+                    encryption_algorithm=serialization.NoEncryption()).decode('utf8'))
```

Comment:
Ideally, there would be support for encrypted private keys, to reduce the risk of exposing the private key - this would add a defense in depth layer, though would add somewhat to configuration complexity.

This would be a nice improvement over the current state.